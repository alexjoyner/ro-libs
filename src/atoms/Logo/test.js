import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './';

describe('Logo component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<Logo logoText="Test" />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
  it('Should contain test text', () => {
    expect(component.contains('Test')).toBe(true);
  });
});
