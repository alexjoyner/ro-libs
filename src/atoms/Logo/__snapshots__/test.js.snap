// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component Should match snapshot 1`] = `
<styled.span
  logoText="Test"
>
  Test
</styled.span>
`;
